src/main/html/index.html

```diff
@@ -8,34 +8,18 @@
   <!--style type="text/css"></style-->
   <link rel="stylesheet" type="text/css" href="lib.css">
   <link href="//cloud.typography.com/7971714/6011752/css/fonts.css" rel="stylesheet" type="text/css" />
-  <link href="min.css" rel="stylesheet" /> </head>
-
+  <link href="min.css" rel="stylesheet" />
+</head>
 <body id="SwaggerUi" class="page-docs" style="zoom: 1;">
-  <!-- Icons --><svg class="hidden" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-      <defs>
-        <symbol ng-repeat="glyph in glyphs" id="icon-copy" mi-view-box="0 0 1024 1024" viewBox="0 0 1024 1024">
-          <title>copy</title>
-          <path ng-repeat="path in glyph.paths" class="path0" mi-d="M640 256v-256h-448l-192 192v576h384v256h640v-768h-384zM192 90.51v101.49h-101.49l101.49-101.49zM64 704v-448h192v-192h320v192l-192 192v256h-320zM576 346.51v101.49h-101.49l101.49-101.49zM960 960h-512v-448h192v-192h320v640z" mi-fill="" mi-opacity="" d="M640 256v-256h-448l-192 192v576h384v256h640v-768h-384zM192 90.51v101.49h-101.49l101.49-101.49zM64 704v-448h192v-192h320v192l-192 192v256h-320zM576 346.51v101.49h-101.49l101.49-101.49zM960 960h-512v-448h192v-192h320v640z"></path>
-        </symbol>
-      </defs>
-    </svg>
-  <header class="site-header" style="display: none;">
-    <nav role="navigation" class="navbar navbar-default">
-      <div class="container">
-        <div class="navbar-header"> <button type="button" data-toggle="collapse" data-target="#navbar-collapse" class="navbar-toggle"><span
-              class="sr-only">Toggle navigation</span><span class="icon-bar"></span><span
-              class="icon-bar"></span><span class="icon-bar"></span></button>
-          <h1 class="navbar-brand"><a href="http://swagger.io"><span>swagger explorer</span></a></h1>
-        </div>
-        <div id="navbar-collapse" class="collapse navbar-collapse">
-          <ul class="nav navbar-nav navbar-left">
-            <li class="li-why"><a href="http://swagger.io" style="font-size: 25px; padding-left: 0px">Swagger
-                  explorer</a></li>
-          </ul>
-        </div>
-      </div>
-    </nav>
-  </header>
+  <!-- Icons -->
+  <svg class="hidden" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <defs>
+      <symbol ng-repeat="glyph in glyphs" id="icon-copy" mi-view-box="0 0 1024 1024" viewBox="0 0 1024 1024">
```

Comment:
yeah no these tags def don't do anything. I thought you added them? The only diff here is that I changed the indents in the file.

Would it be safe to remove the `ng` directives since they don't appear to do anything?